### ISSUE TITLE
Bootstrap EDC API Wrapper Extension

### ISSUE THREAD

**Issue by richardtreier** [sovity/edc-extensions#187]
# Feature
## Description
Initial required setup so we can start working on implementation in parallel.

## Breakdown
- [x] Java API Client with Dummy API Endpoint from #184 
  - [x] Ensure Dummy API Endpoint is tested
  - [x] Ensure Dummy API Endpoint is documented
  - [x] Ensure public facing Java Client API is well documented
- [x] Add README.md explaining each unit, what it is, and why it exists.
- [ ] Merge into main
